### PR TITLE
NetCore: Added documentation on customising the no published content response

### PR DESCRIPTION
**Pull request by AndyButland**
Resolves issue #2366 

Explains how to customise the display of the "no nodes" response when no published content is found in an Umbraco installation, in the .Net Core version of Umbraco being developed.

### CODE REVIEW

File: Extending/No-Published-Content/index.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# No Published Content Page
+
+When Umbraco is first installed, if no starter kit is chosen, there will be no content created in the backoffice, and no published content to display.
+
+Content can also be created, but be unpublished.
+
+If browsing the front-end of the website, when this situation of no published content is found, a custom page is displayed.
+
+This page is served from `/config/splashes/noNodes.aspx`.
```
Comment (bergmania) [inline Extending/No-Published-Content/index.md]:
```suggestion
This page is served from `/config/splashes/NoNodes.cshtml`.
```

Comment (AndyButland) [reply Extending/No-Published-Content/index.md]: Actually, I think this is right as is.  `Index.md` is for the current version, i.e. v8, which does use the `.aspx` page.  The .Net Core version of the documentation page has the extension `-v9.md`

**Comment by sofietoft** [comment]
Thanks a bunch for the PR @AndyButland ! And apologies for taking so long to get back.

We had a meeting in the Documentation team yesterday, and talked about how we should handle these PRs for .NetCore / v9 / vNext - or whatever we should call it 😄 

It's great to see them coming in, and we wanna do what we can to find a place for them in the current Docs! So, based on the discussion in the Core Collaborators slack channel and some other ideas we've had before regarding this, we've come up with a solution, that we all like.

We're going to add the `versionFrom: 9.0.0` YAML tag to all .NET Core specific articles, that way we can merge them in, and it'll be possible to distinguish it from the other articles.
Then, we'll work on some updates to the Our platform, so we can hide all articles with that tag on the Documentation on Our.
The reason we'll be doing that, is to minimize the confusion, and to avoid people thinking that v9 is yet a thing and that articles for .NET Core is actually usable on the current version of Umbraco!

A bit hard to explain, but I hope it makes sense 😁 
And do let me know if you have any further questions.
We'll start working on the changes to Our as soon as possible, so we can get this PR merged! I'm adding the `pending-our-update` label for now.

**Comment by AndyButland** [comment]
Sounds good to me - thanks.  I've already added that `versionFrom` on this page.

**Comment by sofietoft** [comment]
Hi there!
I have an important message for you, regarding this PR:

Yesterday we tried to do some cleaning up and force-pushed to the master branch which now causes some problems with existing forks and local clones. 
We have fixed up all the open PRs so that they are still mergeable now.

With that said, if you've been working on a **fork of the repo** or in a **local clone**, those will not work anymore.
If you wish to push any new changes to this PR, you will need to delete your local clone or fork and create fresh ones.

If you have any local work in progress, then you copy it over from your old fork or clone, following these steps:

**Forks**
1. In the old clone of the old fork, switch to the branch you were working on
2. In the new clone of the new fork, create a new branch with the same name as the old one
3. Copy all the affected files from the old clone to the new clone
4. Repeat steps 1-3 if you've been working with multiple branches
5. You can now safely delete your old fork

**Clones**
1. In the old clone, switch to the branch you were working on
2. Create and checkout a new branch on the new clone, using the same name as the old clone
3. Copy all the affected files from the old clone to the new clone
4. Repeat steps 1-3 if you've been working with multiple branches
5. Delete the old clone of the docs repo

And with that, you should be all set once more.

I do apologize for the inconvenience, and please let me know if you have any questions!
/Sofie

**Comment by dampee** [comment]
Hi @AndyButland 

There is a new docs repo setup here: https://github.com/umbraco/UmbracoCMSDocs.

Do you want to create a new PR there yourself?

**Comment by AndyButland** [comment]
Think it's probably better to wait until there's more in there - either a structure, or whatever you plan to port over from the existing repo - just to see best where this one fits.  As obviously it's really just for one micro-feature.  I'll keep an eye and do so though when it makes sense.

**Comment by AndyButland** [comment]
Closing this PR and will apply to the new repo.

**Pull request closed without merging**